services/QuillLMS/client/app/bundles/Teacher/components/premium/premium_minis/school_pricing_mini.jsx

```diff
@@ -25,8 +25,8 @@ const SchoolPricingMini = ({ plan, premiumFeatureData, showBadges, handleClickPu
         <p>Request Quote or Buy Now</p>
       </div>
       {showBadges && <div className="school-premium-badge-container">
-        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> Onboarding training</div>
-        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> 2 PD workshops</div>
+        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> Quill Academy</div>
+        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> Admin Dashboard</div>
```

Comment:
Since "Quill Academy" is a proper noun it makes sense for it to be title-cased, but I think Admin Dashboard should sentence-cased ("Admin dashboard") to be consistent